leshan-server-core/src/main/java/org/eclipse/leshan/server/bootstrap/DefaultBootstrapHandler.java

```diff
@@ -236,21 +234,19 @@ protected void writeSecurities(final BootstrapSession session, final BootstrapCo
                 @Override
                 public void onResponse(BootstrapWriteResponse response) {
                     if (response.isSuccess()) {
-                        LOG.trace("Bootstrap write {} return code {}", session.getEndpoint(), response.getCode());
+                        LOG.trace("{} receives {} for {}", session, response, writeBootstrapRequest);
                         sessionManager.onSuccess(writeBootstrapRequest);
                         afterWriteSecurities(session, cfg, securityInstancesToWrite, BootstrapAction.CONTINUE);
                     } else {
-                        LOG.debug("Bootstrap write {} return code {}", session.getEndpoint(), response.getCode());
+                        LOG.debug("{} receives {} for {}", session, response, writeBootstrapRequest);
                         BootstrapAction action = sessionManager.onError(writeBootstrapRequest, response);
                         afterWriteSecurities(session, cfg, securityInstancesToWrite, action);
                     }
                 }
             }, new ErrorCallback() {
                 @Override
                 public void onError(Exception e) {
-                    LOG.debug(String.format(
-                            "Error during bootstrap write of security instance %s to foreign peer [ep=%s,id=%s]",
-                            securityInstance, session.getEndpoint(), session.getIdentity()), e);
+                    LOG.debug("Error for {} while sending {} ", session, writeBootstrapRequest, e);
```

Comment:
I think the exception parameter is going to be ignored here. e will be considered as an argument to the string format.

Reply:
I pretty sure it's OK, but I will test this.

Reply:
```java
public static void main(String[] args) {
	LOG.error("My log with params {} {}","param1","param2", new NullPointerException());
}
```
gives
```
17:18:26.659 [main] ERROR test.LOGGER - My log with params param1 param2
java.lang.NullPointerException: null
	at test.LOGGER.main(LOGGER.java:11)
```

So it looks OK

Reply:
ok 🤔 I was sure I've had this issue in the past. anyway, thanks for testing it.

Reply:
Maybe it depends on the slf4j backend :thinking:
I only tested with logback.

Reply:
(it's OK for slf4j-simple too)